app.test.js

```diff
@@ -6,15 +6,6 @@ const environment = process.env.NODE_ENV || 'development'
 const configuration = require('./knexfile')[environment]
 const database = require('knex')(configuration)
 
-describe('Server', () => {
-  describe('init', () => {
-    it('should return a 200 status', async () => {
-      const res = await request(app).get('/')
-      expect(res.status).toBe(200)
```

Comment:
We all agreed that this test was unnecessary based on how our current endpoint structure is set up.

Reply:
I am sure it is not needed anymore